corehq/apps/userreports/views.py

```diff
@@ -1352,7 +1352,7 @@ def rebuild_data_source(request, domain, config_id):
     ))
 
 
-def _number_of_records_to_be_iterated_for_rebuild(datasource_configuration):
+def number_of_records_to_be_processed(datasource_configuration):
```

Comment:
Just a reminder; 
This method was also used [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/tasks.py#L195) and will require rename while merging master.

Reply:
https://github.com/dimagi/commcare-hq/pull/35465/commits/365de982912607259ca2ce843ecd142a06deae7e